OWNERS_ALIASES

```diff
@@ -139,6 +139,8 @@ aliases:
   sig-buildsystem-reviewers:
     - brianmcarey
     - enp0s3
+  # Owns s390x, and ensures that kubevirt can be built on s390x.
```

Comment:
this makes it seem that xpivarc is covered by this comment

Reply:
Hi @dankenigsberg  i can move my name below xpivarc but this will disrupt alphabetical order.

Reply:
can i put something like this?
```
 # Owns s390x, and ensures that kubevirt can be built on s390x.
    - vamsikrishna-siddu
 # end of s390x owners
    - xpivarc
```

Reply:
Hi @dankenigsberg  i have removed the comment and pushed the latest changes. Can you please review it again. Thanks